docs/authentication.md

```diff
@@ -228,21 +228,50 @@ var sdk = new BoxSDK({
 var client = sdk.getBasicClient('YOUR-APP-TOKEN');
 ```
 
-### Anonymous Authentication
+### Client Credentials Grant Authentication
 
-Additionally, you may authenticate as a client without an attached user,
-which can be used to make API calls that do not require a logged-in user (e.g.
-open shared links).  This functionality is only available to approved applications.
+Allows you to obtain an access token by having client credentials and secret with enterprise or user ID,
+which allows you to work using service (anonymous) or user account.
 
-<!-- sample x_auth init_with_anonymous_user -->
+#### Obtaining Service Account token
+
+The [Service Account](https://developer.box.com/guides/getting-started/user-types/service-account//) 
+is separate from the Box accounts of the application developer and the
+enterprise admin of any enterprise that has authorized the app — files stored in that account
+are not accessible in any other account by default, and vice versa.
+To obtain service account you will have to provide enterprise ID with client id and secret:
```

Comment:
minor visual thing - capitalise `id` to match formatting pattern in the rest of the doc.